contentcuration/contentcuration/frontend/shared/views/HelpTooltip.vue

```diff
@@ -1,20 +1,30 @@
 <template>
 
-  <VTooltip maxWidth="150px" v-bind="$attrs" lazy>
-    <template #activator="{ on }">
-      <Icon color="primary" :small="small" v-on="on">
-        {{ icon }}
-      </Icon>
-    </template>
-    <span class="text-xs-center">{{ text }}</span>
-  </VTooltip>
+  <div>
+    <Icon
+      ref="tooltip"
+      :color="$themeTokens.primary"
+      style="font-size:20px;"
+      :icon="icon"
+    />
+
+    <KTooltip
+      reference="tooltip"
+      placement="bottom"
+      :refs="$refs"
+      v-bind="$attrs"
```

Comment:
For `v-bind="$attrs"` please see my questions for the same in _AssessmentEditor.vue_